packages/hardhat-vyper/README.md

```diff
@@ -58,12 +58,23 @@ module.exports = {
 };
 ```
 
-You can also configure multiple versions of the Vyper compiler:
+You can also configure multiple versions of the Vyper compiler together with the compiler settings properties evmVersion (supported starting from Vyper 0.3.8) and optimize (supported starting from Vyper 0.3.10).
```

Comment:
```suggestion
You can also configure multiple versions of the Vyper compiler, including the compiler settings `evmVersion` (available from Vyper 0.3.8) and `optimize` (available from Vyper 0.3.10).
```

Reply:
I changed it, but with a different phrasing